components/devtools/actors/root.rs

```diff
@@ -265,24 +250,12 @@ impl Actor for RootActor {
                         device: DeviceActor::description(),
                     },
                 };
-                let _ = stream.write_json_packet(&msg);
-                ActorMessageStatus::Processed
+                request.reply_final(&msg)?
             },
 
-            _ => {
-                let reply = ErrorResponse {
-                    from: self.name(),
-                    error: "unrecognizedPacketType".to_owned(),
-                    message: format!(
-                        "Actor {} does not recognize the packet type '{}'",
-                        self.name(),
-                        msg_type,
-                    ),
-                };
-                let _ = stream.write_json_packet(&reply);
-                ActorMessageStatus::Ignored
-            },
```

Comment:
this is the one (1) place in the whole devtools impl that was sending `unrecognizedPacketType` correctly for unhandled request types!